### PR TITLE
LPS-88944 We just need one form

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

File: modules/apps/headless-apio/structured-content/structured-content-apio-impl/src/main/java/com/liferay/structured/content/apio/internal/architect/resource/StructuredContentNestedCollectionResource.java [hunk 1]
```diff
@@ -745,6 +747,29 @@ private Long _getStructuredContentId(
...
+		Set<Map.Entry<Locale, String>> map2Entries = map2.entrySet();
+
+		return Stream.concat(
+			map1Entries.stream(), map2Entries.stream()
+		).collect(
+			Collectors.toMap(
+				Map.Entry::getKey, Map.Entry::getValue,
+				(value1, value2) -> value2)
+		);
+	}
```
Comment (victorg1991) [inline modules/apps/headless-apio/structured-content/structured-content-apio-impl/src/main/java/com/liferay/structured/content/apio/internal/architect/resource/StructuredContentNestedCollectionResource.java]: 😍

**Pull request closed without merging**